wp-content/mu-plugins/query-vars.php

```diff
@@ -33,6 +33,7 @@
    *
```

Comment:
Taxonomy terms for employer programs, for use in filtering through the archive scripts